### PR TITLE
refactor!: separate lib and cli

### PR DESCRIPTION
**Pull request by comfysage**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,110 @@
+use textwrap::wrap;
+
+use unicode_width::UnicodeWidthStr;
+
+pub static CAT: &str = "
```
Comment (uncenter) [inline src/lib.rs:5]:
```suggestion
pub static KITTY: &str = "
```
:3

File: src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+	left: "(",
+	right: ")",
+	single_left: "(",
+	single_right: ")",
+	angled_up_right: "⎛",
+	angled_up_left: "⎞",
+	angled_down_right: "⎝",
+	angled_down_left: "⎠",
+};
+pub fn generate_msg(message: &str, format_opts: FormatOptions) -> String {
```
Comment (uncenter) [inline src/lib.rs:58]:
```suggestion
pub fn generate(message: &str, format_opts: FormatOptions) -> String {
```

**Review by uncenter — CHANGES_REQUESTED** [review]

**Review by uncenter — APPROVED** [review]

**Comment by uncenter** [comment]
Thanks!

**Comment by comfysage** [comment]
wait no im stupid

**Comment by comfysage** [comment]
print fn pr in #22

**Pull request merged**